internal/services/recoveryservices/site_recovery_replicated_vm_resource_test.go

```diff
@@ -34,7 +34,26 @@ func TestAccSiteRecoveryReplicatedVm_basic(t *testing.T) {
 	})
 }
 
+func TestAccSiteRecoveryReplicatedVm_withFivePointOhTFOSettings(t *testing.T) {
```

Comment:
We'd want the 5.0 test to just be the old name, since it'll stick around past the 5.0 release

```suggestion
func TestAccSiteRecoveryReplicatedVm_withTFOSettings(t *testing.T) {
```